### PR TITLE
Couple of fixes in istio charts

### PR DESCRIPTION
**Pull request by GoelDeepak**
1. Fix typo in grafana dashboard namespace
2. Creates a destination rule to disabled mTLS for prometheus as it is outside of istio
3. Fixes issue in citadel chart

### CODE REVIEW

**Comment by GoelDeepak** [comment]
cc @gpaul

File: staging/istio/charts/prometheus/templates/destinatiorule.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.istio.io/v1alpha3
```
Comment (joejulian) [inline staging/istio/charts/prometheus/templates/destinatiorule.yaml]: typo in filename

**Comment by GoelDeepak** [comment]
Thanks, @shaneutt for taking care of it

**Comment by gpaul** [comment]
Looks like this can be merged?

**Pull request merged**